README.md

```diff
@@ -1,73 +1,23 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # Project Name
 
-> Description the project.
+Portfolio: Mobile Version
 
+> Description the project.
+> Building a personal portfolio and it's responsive in different platforms.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
```

Comment:
- [ ] It will be nice to update a `Getting Started` section in your `readme file`, where to give explicit steps on how to clone and run your project locally, as this will be of help to non technical users and some of them are potential recruiters.